lib/gem2rpm.rb

```diff
@@ -23,6 +23,11 @@ class DownloadUrlError < Exception; end
       'r'
     end
 
+  def self.show_message(message, obj = nil, out = $stderr)
+    out.puts(format("%s\n\n", message))
```

Comment:
What is the reason to use #format instead of string interpolation, e.g. ```"#{message}\n\n"```. Is there any difference?